packages/mdc-menu/README.md

```diff
@@ -204,7 +204,7 @@ Property | Value Type | Description
 Method Signature | Description
 --- | ---
 `setAnchorCorner(Corner) => void` | Proxies to the menu surface's `setAnchorCorner(Corner)` method.
-`setAnchorMargin(AnchorMargin) => void` | Proxies to the menu surface's `setAnchorMargin(AnchorMargin)` method.
+`setAnchorMargin(MenuPosition) => void` | Proxies to the menu surface's `setAnchorMargin(MenuPosition)` method.
```

Comment:
Is `MenuPosition` the right thing here?

Reply:
Woops. Updated to `Partial<MenuDistance>`. Good catch!